React/my-app/src/screens/book_detail/index.js

```diff
@@ -22,11 +22,9 @@ class BookDetailContainer extends Component {
       author: book.author,
       title: book.title,
       genre: book.genre,
-      year: book.year
+      year: book.year,
+      image: book.imageUrl ? book.imageUrl : defaultImg
```

Comment:
now how you have the defaultProp, you dont need this ternary. Just use `book.imageUrl`
so, if this is undefined or null, the componente will be render the default!